libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -41,14 +49,15 @@ typedef struct OVModel{
     ie_blob_t *input_blob;
 } OVModel;
 
-static const AVClass dnn_openvino_class = {
-    .class_name = "dnn_openvino",
-    .item_name  = av_default_item_name,
-    .option     = NULL,
-    .version    = LIBAVUTIL_VERSION_INT,
-    .category   = AV_CLASS_CATEGORY_FILTER,
+#define OFFSET(x) offsetof(OVContext, x)
+#define FLAGS AV_OPT_FLAG_FILTERING_PARAM
```

Comment:
As I noticed, the AV_OPT_FLAG_VIDEO_PARAM is only used in show help of "./ffmpeg -h full" (fftools/cmdtuils.c: show_hep_children), to show all filters details. While the cmd will not show the options of OpenVINO options, so should I omitted it as above or try to code to show the openvino options in ffmpeg help?

Reply:
just ignore now. It is not designed for our case.